secrets/gitkey.yml.example

```diff
@@ -0,0 +1 @@
+git_key: |
```

Comment:
We've added `secrets/*.yml` to the gitignore file. Do we want these files here as examples as they're named? If we do it might be good to mention them in the README